### PR TITLE
Fix: cannot build native from source

### PR DESCRIPTION
**Pull request by cybertk**
As the latest trax([`36d9e522`][]) changed the directory structure, vot-toolkit is failed to compile it.

Changes:

- Upgrade to build latest trax [`36d9e522`][]
- Add more debugging logs

[`36d9e522`]: https://github.com/votchallenge/trax/tree/36d9e52201ae315a1f025e4184da4e1a48254068

### CODE REVIEW

File: utilities/initialize_native.m [hunk 1]
```diff
@@ -35,19 +35,19 @@ function initialize_native(output_path)
...
-    fullfile(trax_path, 'lib', 'region.c')}, {fullfile(trax_path, 'lib')}, output_path);
+    fullfile(trax_path, 'src', 'region.c')}, {fullfile(trax_path, 'src')}, output_path);
 
 success = success && compile_mex('region_mask', {fullfile(toolkit_path, 'sequence', 'region_mask.cpp'), ...
-    fullfile(trax_path, 'lib', 'region.c')}, {fullfile(trax_path, 'lib')}, output_path);
+    fullfile(trax_path, 'src', 'region.c')}, {fullfile(trax_path, 'src')}, output_path);
 
 success = success && compile_mex('region_convert', {fullfile(toolkit_path, 'sequence', 'region_convert.cpp'), ...
-    fullfile(trax_path, 'lib', 'region.c')}, {fullfile(trax_path, 'lib')}, output_path);
+    fullfile(trax_path, 'srcb', 'region.c')}, {fullfile(trax_path, 'src')}, output_path);
```
Comment (lukacu) [inline utilities/initialize_native.m]: Why is this folder named 'srcb'?

Comment (cybertk) [inline utilities/initialize_native.m]: It should be `src`.

**Comment by cybertk** [comment]
Fixed previous type

**Comment by lukacu** [comment]
Great, thanks.

**Pull request merged**